types/react-router/index.d.ts

```diff
@@ -153,8 +153,17 @@ export const __RouterContext: React.Context<RouteComponentProps>;
 
 export function useLocation<S = H.LocationState>(): H.Location<S>;
 
+export type To =
```

Comment:
I would rename it to something more verbose, however I don't have an idea at this moment. What you think?

Reply:
How about DestinationProp?

Reply:
Can be just Desitnation or DestinationProp as you proposed. I will accept any of these two

Reply:
Done